pipelines/nextflow/modules/genome_metadata/dump_genome_stats.nf

```diff
@@ -27,14 +27,12 @@ process DUMP_GENOME_STATS {
         tuple val(db), path("core_stats.json")
 
     script:
-        def output = "core_stats.json"
         """
-        touch $output
         genome_stats_dumper --host '${server.host}' \
```

Comment:
here and in pyproject to `dump_genome_stats`?

Reply:
Noted for tomorrow.